simpleLooping/createListPowerOfBase.py

```diff
@@ -11,3 +11,21 @@
 # Ex:
 #   createListPowerOfBase(2, 4) # returns [1, 2, 4, 8]
 #   createListPowerOfBase(3, 3) # returns [1, 3, 9]
+from typing import List
+ 
+def createListPowerOfBase(base: int, powerCap: int) -> List[int]:
```

Comment:
I see that you are having 4 if that do unnecessary check or make the code wrong. Should should better check those case like PowerCape < 1 (and even this one is not needed as range(-2) will automatically return an empty list.)